app/Http/Controllers/Api/DeviceController.php

```diff
@@ -142,7 +140,13 @@ public function anyRegisterDevice(RegisterDeviceRequest $request)
             'success' => true,
             'dev_mode' => false,
             'use_gps'  => $default->is_gps_location_source ? true : false,
-            'renew'    => false //@Todo - make possible to enable Temporary. Will be used to re-check BCS projects URL.
+            'renew'    => $device->renew,
+            'user_phone' => $device->user_phone,
```

Comment:
Minor nitpick: I think `user_phone` is a little vague - I think `user_phone_no` (or even just `phone_no`) is more self-documenting

Reply:
Since it is related also with app updates, we will leave with user_phone.